### DISCUSSION TITLE
AList V3 怎么实时查看访客

### DISCUSSION THREAD

**Discussion by maple185** [AlistGo/alist#2434] [Q&A]
环境：WINDOWS 10
版本号/服务端：AList V3

V1版本的时候  在CMD里可以看到访客IP和访问目录，最近换了V3，用什么命令可以像V1那样显示访客IP和访问目录？

谢谢了

**Comment by sha7dowXX** [comment]
js调用外部的

**Comment by NovaShen0724** [comment]
看log
